src/reaper_osara.cpp

```diff
@@ -3956,6 +3956,37 @@ void cmdManageTempoTimeSigMarkers(Command* command) {
 	Main_OnCommand(40256, 0); // Tempo envelope: Insert tempo/time signature change marker at edit cursor...
 }
 
+void cmdSelectItemsUnderEditCursorOnSelectedTracks(Command* command) {
+	vector<MediaItem*> items;
+	for (int i = 0; i < CountTracks(nullptr); ++i) {
+		MediaTrack* track = GetTrack(nullptr, i);
```

Comment:
Should we be skipping tracks that aren't selected? That way, we don't fetch a bunch of items we don't need. So you'd move the isTrackSelected check here and remove it later.